modules/operator-upgrade-change-subscription-channel.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * upgrading/upgrade-operator.adoc
+:_module-type: PROCEDURE
+[id="operator-upgrade-change-subscription-channel_{context}"]
+= Changing subscription channel
+
+[role="_abstract"]
+You can change the update channel for the {product-title-short} Operator by using the {ocp} web console.
```

Comment:
Is it worth providing the CRD way of doing it? @connorgorman

Reply:
Command-line way turns to be quite simple. I added it to my doc https://docs.engineering.redhat.com/display/StackRox/Upgrade+to+4.0#Upgradeto4.0-Operatorupgradefrom3.yto4.0

It's just
```sh
kubectl -n rhacs-operator patch subscriptions.operators.coreos.com rhacs-operator --type=merge --patch='{ "spec": { "channel": "stable" }}'
```